Simple-PHP-IPAM/upgrade.sh

```
@@ -237,10 +225,64 @@ if [[ -n "$CHOWN_BIN" ]]; then
+        echo "Running migrate_db.php..."
+        "$PHP_BIN" "$TARGET_DIR/migrate_db.php" \
+          --from="$current_driver" \
+          --from-dsn="$([[ "$current_driver" == "sqlite" ]] && echo "sqlite:$TARGET_DIR/data/ipam.sqlite" || "$PHP_BIN" -r "echo (require '$TARGET_DIR/config.php')['db_dsn'] ?? '';")" \
+          --to="$target_driver" --to-dsn="$target_dsn" \
+          --to-user="$target_user" --to-pass="$target_pass" \
+          --force || {
+            echo "Driver migration failed. Your original database is unchanged."
+            echo "You can retry manually: php $TARGET_DIR/migrate_db.php --help"
+          }
+
+        if [[ $? -eq 0 ]]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)